cut-n-paste/toolbar-editor/egg-toolbars-model.c

```diff
@@ -27,6 +27,7 @@
 
 #include <unistd.h>
 #include <string.h>
+#include <libxml/xmlsave.h>
```

Comment:
```suggestion
#include <libxml2/globals.h>
#include <libxml2/parser.h>
```

As mentioned in my PR comment, this seems a better include set to me (assuming it actually works, I did *not* test with a real build)

Reply:
@raveit65 it's still here with the `#include <libxml/parser.h>`??  I'll have to check this then, but it looks awfully odd…

Reply:
OK, there must be something I don't get from just looking at the libxml2 sources then.  What do you have with `grep -rF xmlParseFile /usr/include/libxml2`?

Reply:
```
[rave@mother ~]$ grep -rF xmlParseFile ~/fedora-rawhide-x86_64/root/usr/include/libxml2/
/home/rave/fedora-rawhide-x86_64/root/usr/include/libxml2/libxml/parser.h:              xmlParseFile          (const char *filename);
```

Reply:
wait, I see this declaration is guarded by `LIBXML_SAX1_ENABLED`…
What do you get with `grep -rE 'define\s+LIBXML_SAX1_ENABLED' ~/fedora-rawhide-x86_64/root/usr/include/libxml2/`?

Reply:
```
[rave@mother ~]$ grep -rE 'define\s+LIBXML_SAX1_ENABLED' ~/fedora-rawhide-x86_64/root/usr/include/libxml2
/home/rave/fedora-rawhide-x86_64/root/usr/include/libxml2/libxml/xmlversion.h:#define LIBXML_SAX1_ENABLED
```

Reply:
Sorry, actually we need the line above:
`grep -rEB1 'define\s+LIBXML_SAX1_ENABLED' ~/fedora-rawhide-x86_64/root/usr/include/libxml2/`

Reply:
```
[rave@mother ~]$ grep -rEB1 'define\s+LIBXML_SAX1_ENABLED' ~/fedora-rawhide-x86_64/root/usr/include/libxml2/
/home/rave/fedora-rawhide-x86_64/root/usr/include/libxml2/libxml/xmlversion.h-#if 1
/home/rave/fedora-rawhide-x86_64/root/usr/include/libxml2/libxml/xmlversion.h:#define LIBXML_SAX1_ENABLED
```
Btw: i think we got the same issue with mate-calc https://kojipkgs.fedoraproject.org//work/tasks/9110/109319110/build.log

Reply:
I'm at a loss here… if you can't figure this one out yourself, I'll try and get libxml2 2.12 somehow and play with it, but it will be a tad non-trivial.

Reply:
> Btw: i think we got the same issue with mate-calc https://kojipkgs.fedoraproject.org//work/tasks/9110/109319110/build.log

https://github.com/mate-desktop/mate-calc/pull/221

Reply:
@raveit65 I really don't understand, for me my suggestion works just fine, so there ought to be a difference in your setup somehow.

Reply:
Works for me in fedora rawhide https://src.fedoraproject.org/rpms/eom/blob/rawhide/f/eom_0001-fix-building-with-libxml-2.12.0.patch
https://koji.fedoraproject.org/koji/taskinfo?taskID=109398888